management/web_update.py

```diff
@@ -75,13 +76,17 @@ def do_web_update(env):
 	nginx_conf = open(os.path.join(os.path.dirname(__file__), "../conf/nginx-top.conf")).read()
 
 	# Load the templates.
+        template2_nextcloud = ""
```

Comment:
Space/tabs issue. This line should also immediately precede the next set of added lines.